### PR TITLE
Put reviews on movie show page

### PR DESCRIPTION
**Pull request by Jminsky1**
-created migration for reviews table
-set up associations between movies and reviews
-created reviews controller
-created serializers for review and movie and movie_show
-created review list and review tile as children of movieShowContainer in order to put reviews on movie show page
- created a route to reviews

### CODE REVIEW

File: app/models/review.rb [hunk 1]
```diff
@@ -0,0 +1,6 @@
+class Review < ApplicationRecord
+  belongs_to :movie
+
```
Comment (safang99) [inline app/models/review.rb:3]: If a `user has_many :reviews` I'm assuming we will want to include a `belongs_to :user` here

File: db/migrate/20201022143348_create_reviews.rb [hunk 2]
```diff
@@ -0,0 +1,11 @@
+class CreateReviews < ActiveRecord::Migration[5.2]
+  def change
+    create_table :reviews do |t|
+        t.belongs_to :movie, null: false
+        t.integer :rating, null: false
```
Comment (safang99) [inline db/migrate/20201022143348_create_reviews.rb:5]: We will also need to add a `t.belongs_to :user, null: false` if we only want signed in users to write reviews

File: app/serializers/review_serializer.rb [hunk 3]
```diff
@@ -0,0 +1,7 @@
+class ReviewSerializer < ActiveModel::Serializer
+  attributes :id, :body, :rating
+
+  belongs_to :movie
+end
+
+
```
Comment (safang99) [inline app/serializers/review_serializer.rb]: We can get rid of one of the extra empty lines on 6 or 7

File: app/controllers/api/v1/reviews_controller.rb [hunk 4]
```diff
@@ -0,0 +1,5 @@
+class Api::V1::ReviewsController < ApplicationController
+  def show
+    render json: Review.find(params[:id])
+  end
```
Comment (safang99) [inline app/controllers/api/v1/reviews_controller.rb]: Is this controller and controller action being used at all? If not we can remove it

File: app/models/user.rb [hunk 5]
```diff
@@ -1,4 +1,6 @@
 class User < ApplicationRecord
+
+  has_many :reviews
```
Comment (safang99) [inline app/models/user.rb]: We can remove the empty line on 2

**Review by safang99 — CHANGES_REQUESTED** [review]
Some changes regarding the association between a user and a review.  Please reach out if you have any clarifying questions!

File: app/models/review.rb (hunk 1 above)
Comment (Jminsky1) [reply app/models/review.rb:3]: We are planning to add user functionality next week so I removed has_many :reviews from User model to keep consistency.

File: db/migrate/20201022143348_create_reviews.rb (hunk 2 above)
Comment (Jminsky1) [reply db/migrate/20201022143348_create_reviews.rb:5]: We are planning to add user functionality next week so I removed has_many :reviews from User model to keep consistency.

File: app/controllers/api/v1/reviews_controller.rb (hunk 4 above)
Comment (Jminsky1) [reply app/controllers/api/v1/reviews_controller.rb]: Removed

**Pull request merged**